analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Verifier.cs

```diff
@@ -256,40 +355,48 @@ static IEnumerable<TMessage> ReadProtobuf(string path)
         }
 
         public static void VerifyCodeFix(string path,
-                                        string pathToExpected,
-                                        SonarDiagnosticAnalyzer diagnosticAnalyzer,
-                                        SonarCodeFixProvider codeFixProvider,
-                                        IEnumerable<ParseOptions> options = null,
-                                        OutputKind outputKind = OutputKind.DynamicallyLinkedLibrary,
-                                        IEnumerable<MetadataReference> additionalReferences = null) =>
+                                         string pathToExpected,
```

Comment:
Line 323 above: `VerifyNoIssueReportedFromCSharp9InTest` seems to became unused and can be removed now.